### ISSUE TITLE
Creating edges without creating new vertexes

### ISSUE THREAD

**Issue by gustavocarmo** [orientechnologies/orientdb-etl#58]
Hi.

I have created all my vertexes using ETL. These are user vertexes (Which let's call it MyUser).
Now, in a separate ETL operation I would like to create edges between them, just like in a communication network:
A->B (#messages sent as a property of the edge)

I tried this:

{
  "source": { "file": { "path": "F:/OrientDB/bin/data/rels.csv" } },
  "extractor": { "row": {} },
  "transformers": [
    { "csv": {} },
    { "vertex": { "class": "MyUser" } },
    { "edge": { "class": "SENT_MESSAGE",
                "joinFieldName": "end",
                "lookup": "MyUser.id",
                "direction": "out",
                "unresolvedLinkAction":"CREATE"
            }
        }
  ],
  "loader": {
    "orientdb": {
       "dbURL": "remote:localhost/databases/MyDB",
       "dbType": "graph",
       "classes": [
         {"name": "MyUser", "extends": "V"},
         {"name": "SENT_MESSAGE", "extends": "E"}
       ]
    }
  }
}

But seems like I have the wrong understanding with that "Vertex" in the beginning. If I don't use that vertex in the transformer it seems like it is also not right, as ETL does not know what nodes to actually use.

Basically:

Nodes
"id","name","label"
1,"Gustavo","Person"
2,"Vikram","Person"

Edges
"start","end"
"2","1"

Then create a simple Gustavo->Vikram relashionship using ETL.

How can I accomplish that?

**Comment by lvca** [comment]
Take a look at: http://orientdb.com/docs/last/Import-from-CSV-to-a-Graph.html. For questions, please don't open new issues, but rather write to the Community Group where more than 2k users can answer to your question:

https://groups.google.com/forum/#!forum/orient-database
